models/s3.js

```diff
@@ -0,0 +1,19 @@
+const aws = require('aws-sdk');
+
+const s3 = new aws.S3({
+	AccessKeyID: process.env.AWS_ACCESS_KEY_ID,
```

Comment:
You don't need to set these there. If the environment variables are named correctly they get picked up automatically by the SDK config.

Reply:
Ah, did not know this
Fixed